aa_intercom/utils.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+
+import pytz
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.utils.timezone import make_aware, now
+from intercom import errors
+from intercom.client import Client
+
+intercom = Client(personal_access_token=settings.INTERCOM_API_ACCESS_TOKEN)
+
+
+def upload_intercom_user(obj_id):
+    """Creates or updates single user account on intercom
```

Comment:
Please move it to new line, alignt with """ or move to one line w/o spaces between. """ and text.